libs/application/core/src/lib/formBuilders.ts

```diff
@@ -93,6 +93,7 @@ export function buildDataProviderItem(data: {
   title: MessageDescriptor | string
   subTitle?: MessageDescriptor | string
   source?: string
+  parameters?: any
```

Comment:
Not used?

This is probably not the right place to define data provider parameters, since this goes through the client. We need some way to define the payment details on the server-side.

The RFC proposed having this configurable in the template as part of some state. I understand you moved it to the template API and a data provider (to make it easy to show the amount in the form), but then we'll need a way to configure data provider parameters in the template somehow. Though this could be useful in other places too.

Did you encounter issues describing payments in the template, with a resolver to get payment details for the UI?

Reply:
We did experience some problems with showing the payment details amount in the form.
But I suppose the biggest problem is that this payable application is being rushed very much.
We will discuss this and see if we can solve it in a different way.

Reply:
yes, not used.. probably left in there from some time ago.